apps/bare-expo/android/app/build.gradle

```diff
@@ -232,43 +232,24 @@ task copyDownloadableDepsToLibs(type: Copy) {
 }
 
 class ExecuteSetupAndroidProject implements Plugin<Project> {
-    @Override
-    void apply(Project target) {
-        target.exec {
-            workingDir '../..'
-            commandLine "./scripts/setup-android-project.sh"
-        }
-    }
-}
-
-apply plugin: ExecuteSetupAndroidProject
-
-import org.apache.tools.ant.taskdefs.condition.Os
-
-class GenerateDynamicMacrosPlugin implements Plugin<Project> {
     @Override
     void apply(Project target) {
         target.exec {
             def configuration = target.gradle.startParameter.taskNames.any {
                 it.toLowerCase().contains("release")
             } ? "release" : "debug"
 
-            workingDir '../../../../tools/bin'
-
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd.exe', '/c', "expotools.js android-generate-dynamic-macros --configuration ${configuration}"
```

Comment:
Are you sure we can remove this? 😅

Reply:
I think so, cause the other script was invoked in the same way on both systems 🤷